### DISCUSSION TITLE
Stripe Checkout Plugin

### DISCUSSION THREAD

**Discussion by rpmcoding** [Budibase/budibase#7934] [Ideas]
@melohagan thanks for creating the plugin for Stripe subscriptions.  It would be amazing if you or someone else is able to create a plugin for Stripe Checkout.

**Comment by melohagan** [comment]
Hey @rpmcoding 

You're welcome. I can pick that one up next!

**Reply by rpmcoding** [reply]
@melohagan Belated thanks for this.

**Comment by kerbersystems** [comment]
Do you have an example on how to use the Create function? I may be missing something but I don't understand how to pass line items to the call.
